tbtc-ts/src/bitcoin.ts

```diff
@@ -83,6 +100,90 @@ export type UnspentTransactionOutput = TransactionOutpoint & {
   value: number
 }
 
+/**
+ * Information about important parts of a transaction, including inputs and
+ * outputs.
+ */
+export interface TxInfo {
+  /**
+   * Transaction's version.
+   */
+  txVersion: string
+
+  /**
+   * All transaction's inputs, prepended by the number of transaction inputs.
+   */
+  txInputVector: string
+
+  /**
+   * All transaction's outputs prepended by the number of transaction outputs.
+   */
+  txOutputVector: string
+
+  /**
+   * Transaction's locktime.
+   */
+  txLocktime: string
+}
+
+/**
+ * Proof that a given transaction is included in the Bitcoin blockchain.
+ */
+export interface TxProof {
+  /**
+   * The merkle proof of transaction inclusion in a block.
+   */
+  merkleProof: string
+
+  /**
+   * Transaction index in the block (0-indexed).
+   */
+  txIndexInBlock: number
+
+  /**
+   * Single byte-string of 80-byte block headers, lowest height first.
+   */
+  bitcoinHeaders: string
+}
+
+/**
+ * Sweep transaction data for verifying the inclusion of a transaction in the
+ * blockchain and processing inputs and outputs.
+ */
+export interface SweepData {
```

Comment:
If we decide to follow the approach proposed in https://github.com/keep-network/tbtc-v2/pull/86#discussion_r819531383 we won't need it.

Reply:
Done.